### DISCUSSION TITLE
aws-cdk and cdk-nag peer dependecy.

### DISCUSSION THREAD

**Discussion by floydspace** [aws/aws-pdk#581] [Q&A] [answered]
I'm wondering if @aws/pdk/monorepo can be not dependent on aws-cdk and cdk-nag? I'm currently using it for open-source monorepo libs project and it's kind of weird that I have cdk in dependencies.

**Comment by agdimech** [comment, marked as answer]
Unfortunately not as we are bundling all code within the PDK into a single distributable. We used to publish the individual packages however our users constantly ran into version mismatch errors when one pdk package didn’t match another and that is why we changed to a single distributable.

Is this causing any issues for you by having it included?

**Reply by floydspace** [reply]
Hi @agdimech, It does not really cause issues. My inner perfectionist simply suffers.
So do what you think is the best, but version mismatch should not be an issue if properly manage dependencies and lock files
Anyway, thank you for your response.
